plugins/kubernetes/app/decorators/build_decorator.rb

```diff
@@ -5,4 +5,31 @@
   def project_name
     @project_name ||= project.name.parameterize('-')
   end
+
+  def import_kubernetes_roles
+    # TODO : Build -> KubernetesRoles association
+    project.roles.destroy_all
```

Comment:
Sorry @jonmoter, you were caught by surprise here. We were having some thoughts around how to refresh the Kubernetes roles. I'll write you an email explaining some new ideas we're experimenting with. I'll C.c. the guys as well and we can discuss it there.